src/custom/utils/userAgent.ts

```diff
@@ -0,0 +1,11 @@
+import { userAgent } from '@src/utils/userAgent'
+
+export * from '@src/utils/userAgent'
+
+function getBrowserMajorVersion() {
+  const major = userAgent.browser.version?.split('.')[0]
+
+  return major ? parseInt(major) : undefined
```

Comment:
What if `major` is not parseable? For example, you get `76-alpha`
I expect it would be fine, but just in case...

Reply:
Heheh, i think we are still safe 😂

<img width="749" alt="image" src="https://user-images.githubusercontent.com/2352112/183381896-72abb550-f891-4d8d-bf1a-7526d617b9a8.png">

Reply:
Also, is not semversioning the 76-alpha (you cannot add that in the major version)

Reply:
JS is sometimes a bit crazy :P

Reply:
`parseInt('76-alpha') === 76` ??????????

![](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.pngmart.com%2Ffiles%2F11%2FFlipping-Tables-PNG-Image.png&f=1&nofb=1)